.github/workflows/kubeapps.yaml

```diff
@@ -7,6 +7,7 @@ on:
   push:
     branches:
       - main
+      - optimize-ci-pipeline
```

Comment:
Guessing this needs to be removed before you land? (assuming it's just for testing this branch?)

Reply:
Yeah, you're right